Source/Core/DolphinQt/CheatsManager.cpp

```diff
@@ -494,6 +537,29 @@ void CheatsManager::NextSearch()
   Update();
 }
 
+static QString GetResultString(const Result& result)
+{
+  if (!PowerPC::HostIsRAMAddress(result.address))
+  {
+    return QStringLiteral("---");
+  }
+  switch (result.type)
```

Comment:
Is this `switch` an exhaustive list of values? MSVC doesn't think so, and errors out because there may not be a return value in all cases.

Reply:
It is exhaustive. MSVC does that sometimes.